baseline-error-prone/src/test/java/com/palantir/baseline/errorprone/StreamFlatmapOptionalTest.java

```diff
@@ -147,6 +147,119 @@ public void test() {
                 .doTest();
     }
 
+    @Test
+    public void testGuavaStreams() {
```

Comment:
These should just be additional cases added to `test`.

I don't think we need to duplicate everything. Many of these cases are testing particular structures that we don't need to re-test for `Streams::stream`.